components/enigmas/DayOnePuzzle1/DayOnePuzzle1.tsx

```diff
@@ -0,0 +1,14 @@
+import styles from './DayOnePuzzle1.module.scss';
+import PuzzleLayout from '../PuzzleLayout';
+
+export default function DayOnePuzzle1() {
+  return (
+    <PuzzleLayout bgStyle={styles.bg}>
+      <div className={styles.text}>
+        <p>
```

Comment:
```suggestion
        <p className={styles.text}>
```

dá pra colocar a class no `<p>`, pra evitar usar mais uma `<div>`